src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -10588,6 +10588,87 @@ public void InsertDeleteMethod_Active()
 
         #endregion
 
+        #region Records
+
+        [Fact]
+        public void Record()
```

Comment:
Would appreciate feedback on whether these are even valid, let alone if they cover enough.

Reply:
They are valid, but I don't think they add much value. The question is - are property initializers special in records vs. classes?

---
In reply to: [595704039](https://github.com/dotnet/roslyn/pull/51926#discussion_r595704039) [](ancestors = 595704039)

Reply:
> are property initializers special in records vs. classes?

Not as far as I know, except in how they affect contents of the synthesized members.